### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by orouz**
fixes:
- [x] [Update README](https://github.com/build-security/kibana/pull/43#discussion_r773748046)

from:
- https://github.com/elastic/security-team/issues/2580

we'll probably iterate on our README as we move forward

### CODE REVIEW

File: x-pack/plugins/cloud_security_posture/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# CSP
+# Cloud Security Posture Kibana Plugin
 
-A Kibana plugin
+Cloud Posture automates the identification and remediation of risks across cloud infrastructures
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/README.md:3]: @tehilabuildsecurity 

this is [taken](https://www.crowdstrike.com/cybersecurity-101/cloud-security/cloud-security-posture-management-cspm/#:~:text=automates%20the%20identification%20and%20remediation%20of%20risks%20across%20cloud%20infrastructures) from online as it describes it better than i could. do we want something else in here?

**Review by ari-aviran — APPROVED** [review]

File: x-pack/plugins/cloud_security_posture/README.md (hunk 1 above)
Comment (tehilashn) [reply x-pack/plugins/cloud_security_posture/README.md:3]: @orouz - I think it should be more informative. similar to what security did https://github.com/build-security/kibana/tree/main/x-pack/plugins/security. 
I think we should consult with product on the content here

**Comment by amar-rama-elastic** [comment]
@orouz  @tehilashn  two questions
1. who is the audience for this readme (developers, kibana admins or end users?)
2. what is the purpose of this readme? (informative or installation instructions)

The example above for instance is pretty technical and not something product would be good at writing up.

**Comment by orouz** [comment]
> 1. who is the audience for this readme (developers, kibana admins or end users?)

it's aimed for kibana developers, but technically, anyone with access to the `README`. 

> The example above for instance is pretty technical and not something product would be good at writing up.

I agree. these files usually have a `title`, `description` and then some technical notes. 
i was hoping to get your approval on the description part: 

> Cloud Posture automates the identification and remediation of risks across cloud infrastructures

which i grabbed from somewhere online. so we might want rephrase it ourselves. 

@amar-rama-elastic

**Comment by amar-rama-elastic** [comment]
@orouz this works. thank you. ++

**Pull request merged**